lua/terrortown/lang/tr.lua

```diff
@@ -3,7 +3,7 @@
 local L = LANG.CreateLanguage("tr")
 
 -- Compatibility language name that might be removed soon.
--- the alias name is based on the original TTT language name
```

Comment:
is there a reason you removed that line?

Reply:
When I first started translating the file, the Turkish language did not exist in the original TTT. I translated the original later. I can add it back if it is neccesary.

Reply:
yes, please keep it there